app/views/users/show.html.erb

```diff
@@ -1,15 +1,15 @@
 <% if current_user %>
 
 <section class="podcast_following center-search">
-  <h1><%= @user.following_podcasts.length %> Following</h1>
+  <h1>Shows you are Following: <%= @user.following_podcasts.length %></h1>
   <% @user.following_podcasts.each do |podcast| %>
     <section id="follow">
-      <%= link_to podcast.name, "/podcasts/#{podcast.id}" %>
+      <%= button_to  podcast.name , podcast_path(podcast.id), method: 'get', class: "btn btn-outline btn-xl js-scroll-trigger" %>
```

Comment:
I made this the normal button we have been using and changed the tests to match.